nifi-docs/src/main/asciidoc/administration-guide.adoc

```diff
@@ -76,6 +76,35 @@ NOTE: For security purposes, when no security configuration is provided NiFi wil
 
 See the <<system_properties>> section of this guide for more information about configuring NiFi repositories and configuration files.
 
+== Build a Custom Distribution
+
+The binary build of Apache NiFi that is provided by the Apache mirrors does not contain every NAR file that is part of the official release. This is due to size constraints imposed by the mirrors to reduce the expenses associated with hosting such a large project. The following Maven profiles are available to administrators that want to simplify the process of building in-house distributions of Apache NiFi that contain the additional functionality that is normally excluded from the official binary releases.
+
```

Comment:
Suggest adding:

[options="header,footer"]

so that table column titles are bolded.